app/pages/QuestionPage.scala

```diff
@@ -43,4 +44,6 @@ trait QuestionPage[+A] extends Page with Gettable[A] with Settable[A] {
     request.userAnswers.submissionFailures.collect {
       case error if possibleErrors.map(_.code).contains(error.errorType) && !error.hasBeenFixed => SubmissionError(error.errorType)
     }
+
+  def value[T >: A](implicit request: DataRequest[_], reads: Reads[T]): Option[T] = request.userAnswers.get[T](this)
```

Comment:
This is great but it'd be good to be consistent everywhere - either implement this everywhere we're calling `request.userAnswers.get` (probably not worth it considering how much of a chore that'd be), or leave this for now.

Reply:
Or at least use it everywhere you've touched in this PR - e.g. in DestinationTypePage.